.github/workflows/build-and-push.yaml

```diff
@@ -0,0 +1,108 @@
+name: Build and Push
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-*'
+
+# env:
+  ## SET_NAME - overrides the docker image name (name is repo name with docker- prefix stripped)
+
+jobs:
+  pre:
+    runs-on: ubuntu-latest
+    outputs:
+      image: ${{ steps.set.outputs.image }}
+      kubectlVersion: ${{ steps.version.output}}
+    steps:
+      - id: set
+        run: |
+          # Computed org/repo name, docker- prefix is stripped off
+          export DEFAULT_IMAGE=$(echo -n '${{ github.event.organization.login }}/'; echo '${{ github.event.repository.name }}' | sed 's/^docker-//' | tr '[:upper:]' '[:lower:]')
+          echo "::set-output name=image::${SET_NAME:-$DEFAULT_IMAGE}"
+      - id: version
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          # Computed version (pick tag's stripped semver, i.e. only vX.Y.Z)
+          KUBECTL_VERSION=$(echo '${{ github.ref }}' | sed -E -e 's/refs\/tags\///' -e 's/(v[0-9]+.[0-9]+.[0-9]+).*/\1/')
```

Comment:
I've used to `;` like 
```
sed -E -e 's/refs\/tags\///;s/(v[0-9]+.[0-9]+.[0-9]+).*/\1/'
```
Didn't know about multiple `-e`, it's a more cleaner way!